src/overlays/actors/ovl_Dm_Zl/z_dm_zl.c

```diff
@@ -171,29 +176,29 @@ void DmZl_UpdateFace(DmZl* this) {
             break;
         case 1:
             if (this->blinkTimer == 0) {
-                this->eyeTextureIndex1 = this->eyeTextureIndex2 = 2;
+                this->eyeTextureIndex1 = this->eyeTextureIndex2 = ZELDA_EYE_CLOSED;
             }
             break;
         case 2:
             if (this->blinkTimer == 0) {
-                this->eyeTextureIndex1 = 5;
-                this->eyeTextureIndex2 = 6;
+                this->eyeTextureIndex1 = ZELDA_EYE_OPEN_LOOKING_LEFT;
+                this->eyeTextureIndex2 = ZELDA_EYE_OPEN_LOOKING_RIGHT;
```

Comment:
These enums make this much more readable, thanks. Can we can figure out what these two indices are? Are they left and right eye?

Reply:
I'm still not sure which is which, is that zelda looking left or zelda looking right? I still struggle to understand how DL handle textures, so not sure which is flipped and which is not, both say NOMIRROR...?
(also why does it load the same tlut twice...?)
from the HeadDL
```c
   gsDPSetTextureLUT(G_TT_RGBA16),
    gsDPLoadTLUT_pal256(gDmZl4EyeTLUT),
    gsDPLoadTextureBlock(0x08000000, G_IM_FMT_CI, G_IM_SIZ_8b, 32, 32, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR |
                         G_TX_CLAMP, 5, 5, G_TX_NOLOD, G_TX_NOLOD),
    gsDPSetTileSize(G_TX_RENDERTILE, 0, 0, 0x007C, 0x007C),
    gsSPVertex(&object_zl4Vtx_0065B0[572], 19, 0),
    gsSP2Triangles(0, 1, 2, 0, 3, 2, 4, 0),
    gsSP2Triangles(3, 5, 0, 0, 0, 2, 3, 0),
    gsDPPipeSync(),
    gsDPSetTextureLUT(G_TT_RGBA16),
    gsDPLoadTLUT_pal256(gDmZl4EyeTLUT),
    gsDPLoadTextureBlock(0x09000000, G_IM_FMT_CI, G_IM_SIZ_8b, 32, 32, 0, G_TX_NOMIRROR | G_TX_CLAMP, G_TX_NOMIRROR |
                         G_TX_CLAMP, 5, 5, G_TX_NOLOD, G_TX_NOLOD),
    gsDPSetTileSize(G_TX_RENDERTILE, 0, 0, 0x007C, 0x007C),
    gsSP2Triangles(6, 7, 8, 0, 8, 9, 6, 0),
    gsSP2Triangles(10, 7, 6, 0, 7, 11, 8, 0),
    gsDPPipeSync(),
    gsDPSetTextureLUT(G_TT_RGBA16),

```

Reply:
I'm fairly sure that what we call eyeTextureIndex2 is the right eye, and the eyeTextureIndex1 is the left one. You can see the them use load the texture for eyeTextureIndex2 into segment 8 here:
```c
gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(sEyeTextures[this->eyeTextureIndex2]));
```

And when I put a texture into that segment in Z64Utils, her right eye gets "filled in" while her left eye remains "empty":
![image](https://user-images.githubusercontent.com/12245827/172983942-02b23759-f89e-49f2-9b3c-30dd47a27bc6.png)

A similar thing happens to the left eye if you put a texture in segment 9, which the code uses for eyeTextureIndex1. I'm not 100% certain how this DList is handling the texture either, but just experimentally, I think we can safely call these `leftEyeIndex` and `rightEyeIndex`, or something similar.

Reply:
The other way would have been to just lock this in an emulator.